standards/cloud_defaults.md

```diff
@@ -0,0 +1,9 @@
+# Cloud Architecture Defaults
+
+The following standards outline the recommended configurations and practices for cloud-based services to ensure optimal security, scalability, and efficiency in line with industry best practices.
```

Comment:
@diazGeoff let's add the diagram given by @rmarfil3 here so it serves as a visual guide for our cloud defaults